### ISSUE TITLE
Abort trap: 6

### ISSUE THREAD

**Issue by ghost** [mattanger/ckb-next#46]
I was doing a clean installation (_all previous ckb files including daemon were removed manually_) with `./quickinstall` on sierra .2 and at the end got this:
```
Start ckb now [Y/n]? y
./quickinstall: line 228: 31827 Abort trap: 6           $APP --close > /dev/null 2>&1
```
I feel like there's a bug in `src/ckb/main.cpp: isRunning()`, look at this part of `main()`:
```cpp
case CommandLineClose:
        // If launched with --close, kill existing app
        if (isRunning("Close"))
            printf("Asking existing instance to close.\n");
        else
            printf("ckb is not running.\n");
        return 0;
```
This is supposed to print the statement in the `else` close, isn't it? Instead it doesn't print any of them. From what I googled about this error, it happens for the same reason when segfault does - no access to the particular chunk of memory allowed.

**Comment by ghost** [comment]
Okay, so I ran it the other day and it prints either of these two printfs without any error if I remove the output redirection from `quickinstall`. Without removing it there's no error either. Strange.

**Comment by ghost** [comment]
So, I finally figured it out :)
This happens if one builds ckb with Homebrew's Qt, but because it has a bug (number 56814 on their tracker), when ckb launches it tries to use the linked frameworks from Homebrew's version and finds the same definitions in official Qt installation. This is scheduled to be fixed in Qt 5.10 only.
